payments/tests/test_payments_queries.py

```diff
@@ -0,0 +1,255 @@
+from decimal import Decimal
+
+import pytest
+
+from berth_reservations.tests.utils import assert_not_enough_permissions
+from payments.schema import BerthPriceGroupNode, BerthProductNode
+from payments.tests.factories import BerthProductFactory
+from resources.schema import HarborNode
+from utils.relay import to_global_id
+
+BERTH_PRICE_GROUPS_QUERY = """
+query BERTH_PRICE_GROUPS {
+    berthPriceGroups {
+        edges {
+            node {
+                id
+                name
+                defaultProduct {
+                    id
+                    priceValue
+                    priceUnit
+                    taxPercentage
+                    harbor {
+                        properties {
+                            name
+                        }
+                    }
+                }
+                products {
+                    edges {
+                        node {
+                            id
+                            priceValue
+                            priceUnit
+                            taxPercentage
+                            harbor {
+                                properties {
+                                    name
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
+"""
+
+
+@pytest.mark.parametrize(
+    "api_client",
+    ["berth_supervisor", "berth_handler", "berth_services"],
+    indirect=True,
+)
+def test_get_berth_price_groups(api_client, berth_price_group):
+    product = BerthProductFactory(price_group=berth_price_group, harbor=None)
+    product_dict = {
+        "id": to_global_id(BerthProductNode, product.id),
+        "priceValue": str(Decimal(product.price_value).quantize(Decimal("1.00"))),
```

Comment:
I think all of these could be just `str(product.price_value)`, since `product.price_value` is already a `Decimal` and simply casting it to string will preserve the decimal points

Reply:
The problem is that the returned value comes with 2 decimals (I guess because the `DecimalField` has a fixed number of `decimal_places=2`?).

<img width="576" alt="image" src="https://user-images.githubusercontent.com/15201480/82637370-4dae5500-9c0d-11ea-9abd-2ec3e6f48c86.png">

So by doing the `Decimal(...).quantize(...)` it creates a decimal with 2 decimal places.

Another option would be to do `round(Decimal(...), 2)`

Reply:
Solved by changing how the `AbstractBaseProductFactory` generates the `price_value`


[payments/tests/factories.py#L23](https://github.com/City-of-Helsinki/berth-reservations/pull/211/files#diff-b5be5d575a3ebb66c94f3efd538238c5R23)
```python
price_value = factory.LazyFunction(
    lambda: round(Decimal(random.uniform(1, 999)), 2)
)
```

Reply:
so, this should now work with just `str(product.price_value)`?

Reply:
Yup! It has [already been fixed](https://github.com/City-of-Helsinki/berth-reservations/pull/211/files#diff-ba338259a0da5db71cb5e6bb5bd5b724R69)